Das Schwarze Auge (TDE) 5 OFFICIAL by Ulisses/translation.json

```diff
@@ -468,6 +469,8 @@
     "zeit": "Zeit",
     "passende-kleidung": "Passende Kleidung",
     "geeignete-geraetschaften": "Geeignete Gerätschaften",
+    "peitschen": "Peitschen",
```

Comment:
why two keys for Peitschen? wouldn't one be sufficient? 
i suggest to delete line 473 and correct the html file if the key with a majuscule was used.

Reply:
in fact the line 473 is useless because not used in the html file